### PR TITLE
Footer finished

### PR DESCRIPTION
**Pull request by maalyhesham**
create footer section #1

### CODE REVIEW

**Review by Yazan-kass — COMMENTED** [review]
If you can turn it responsive

**Review by Yazan-kass — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
-      </header>
+    <div >
+      <Routes>
+            <Route path='/' element={<footer/>} />
+            {/* <Route path='/about'  />
+            <Route path='/' />
+            <Route path='/' />
+            <Route path='/'  />
+            <Route path='/'  />
+            <Route path='/'  /> */}
```
Comment (Mossa-99) [inline src/App.tsx:16]: commented code

File: src/components/Footer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import FacebookIcon from '@mui/icons-material/Facebook';
+import { Container } from '@mui/system';
+
+
+
+ const Footer: React.FC= () => {
+  return (
+    <><Container>
+      <SingFooter/>
+    <FooterStyle>
```
Comment (Mossa-99) [inline src/components/Footer/index.tsx:14]: please use prettier for code formatting
https://www.youtube.com/watch?v=h3PJjP0nE98

File: src/components/Footer/index.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+        <Parent>
+         <h1>INFO</h1>
+        <List>
+            <AnchorLink to='/GIFT VOUCHERS' > GIFT VOUCHERS </AnchorLink>
+            <AnchorLink to='/SIZE GUIDE' > SIZE GUIDE </AnchorLink>
+            <AnchorLink to='/CAREERS' >CAREERS </AnchorLink>
+            <AnchorLink to='/ABOUT US' > About us  </AnchorLink>
+            <AnchorLink to='/GUIDE' > GUIDE  </AnchorLink>
+            <AnchorLink to='/FAQ'> LEGAL Policies  </AnchorLink>
```
Comment (Mossa-99) [inline src/components/Footer/index.tsx:36]: Please remove spaces in the text, you can add spaces and margins using css

File: src/components/SingFooter/index.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { SingFooter } from "../Footer/style";
+import {InboxInput } from "../Footer/style";
+function SingFootersection() {
```
Comment (Mossa-99) [inline src/components/SingFooter/index.tsx:3]:
```suggestion
function SignInFooter() {
```
and if the footer is the same for the whole app, you can name it just ``Footer``

**Review by Mossa-99 — CHANGES_REQUESTED** [review]

**Comment by Yazan-kass** [comment]
Done

**Pull request closed without merging**